File: modul4/Python/PRAK401_2310817210017_MUHAMMADNURHUDABAIHAQI.py
while True:
    input_str = input("")
    
    kelipatan, simbol = map(str, input_str.split())
    kelipatan = int(kelipatan)

    for i in range(1, 51):
        if i % kelipatan == 0:
            print(f"{simbol} ", end="")
        else:
            print(f"{i} ", end="")

    print()

File: modul4/C/PRAK401_2310817210017_MUHAMMADNURHUDABAIHAQI.c
#include <stdio.h>

int main() {
    int kelipatan;
    char simbol;
    while(1){
    printf(" ");
    scanf("%d %c", &kelipatan, &simbol);

    for (int i = 1; i <= 50; i++) {
        if (i % kelipatan == 0) {
            printf("%c ", simbol);
        
        } else {
            printf("%d ", i);
        }
        
    }
    printf("\n");
    }
    return 0;
}
